resources/views/entities/pages/relations/update.blade.php

```diff
@@ -67,14 +67,16 @@
     @if(!empty($from))
         <input type="hidden" name="from" value="{{ $from }}" />
     @endif
-
     {!! Form::hidden('owner_id', $entity->id) !!}
-    {!! Form::close() !!}
+    {!! Form::hidden('option', request()->option) !!}
+    {!! Form::hidden('mode', request()->mode) !!}
 
+    {!! Form::close() !!}
     {!! Form::open([
         'method' => 'DELETE',
-        'route' => ['entities.relations.destroy', $entity->id, $relation->id],
-        'id' => 'delete-relation-' . $relation->id]) !!}
+        'route' => ['entities.relations.destroy', 'entity' => $entity->id, 'relation' => $relation->id, 'mode' => request()->mode, 'option' => request()->option],
```

Comment:
Same here, 2x`get()` means that in the future we can also give a second, fallback value if we wish to build one in